### ISSUE TITLE
Errors in french version

### ISSUE THREAD

**Issue by biozic** [cgnieder/ghsystem#8]
When using the french language, calling this precautionary statement:
```tex
\ghs{P}{305+351+338}
```
I get this error:
```
Undefined control sequence.
\c_ghsystem_french_colon_tl ->:
                               
l.6 \ghs{P}{305+351+338}
```
No errors in English nor German.

Version 4.8b 2020/01/16.

**Comment by cgnieder** [comment]
I get no errors with this code and an up to date TeX distribution:

```latex
\documentclass{article}
\usepackage[T1]{fontenc}
\usepackage[french]{babel}
\usepackage{ghsystem}

\begin{document}

\ghs{P}{305+351+338}

\end{document}
```

**Comment by biozic** [comment]
Sorry, I've just found that the only difference is that I compile the code with LuaLaTeX or XeLaTex. It only works for me with PDFTeX.
My setup is MacTeX 2019, up to date.

**Comment by cgnieder** [comment]
Now I understand. With LuaLaTeX and XeLaTeX `babel` (or `polyglossia`) doesn't make `:` active because ther are aother ways to add the inter-character space… `ghsystem` doesn't take this into account.
I'll fix it.

**Comment by cgnieder** [comment]
Fixed in eee78ebe5a0737b9cf61dd3d234b3455fc2fb75c

**Comment by biozic** [comment]
Great. Thanks!
